extension-reference/mobile/build-your-own-extension/events/identity/events-handled-by-the-identity-service.md

```diff
@@ -1,39 +1,135 @@
+
+```json
+{
+    "baseurl": "https://example.com/path?query=parameter1&other=parameter2"
+}
+```
 
-## Visitor MID Request
+### Event Data Example
 
-This event is generated by customers who use the `visitorMarketingCloudID` API.
+**Sync identifiers request**
+
+```json
+{
+    "visitoridentifiers": {
+        "idtype1" : "id1",
+        "idtype2" : "id2"
+    },
+    "authenticationstate" : 1,
+    "forcesync" : false,
+    "issyncevent" : true
+}
+```
+
+### Event Data Example
+
+**Experience Cloud ID request**
+
+There are no key-value pairs for this event.
+
+### Event Data Example
+
+**Identifiers request**
 
 There are no key-value pairs for this event.
 
-### Visitor IDS Sync Request
 
-This event is generated by customers who use the `visitorSyncIdentifiers` API.
 
-#### Data Payload Definition
+## Generic Identity Request Content
+
+This event is used to set identifiers in the Identity extension and it is generated by:
+
+- Calling the MobileCore (Android) / ACPCore (iOS) `setAdvertisingIdentifier` API
+- Calling the MobileCore (Android) / ACPCore (iOS)  `setPushIdentifier` API
+
+### Event Details
+
+| **Event Type**                       | **Event Source**                      | **Paired** | **Paired Event**                                             |
+| :----------------------------------- | :------------------------------------ | :--------- | :----------------------------------------------------------- |
+| com.adobe.eventType.generic.identity | com.adobe.eventSource.requestIdentity | Yes        | [Link To The Paired Event](../link/to/the/eventpage/andsection) |
+
+### Data Payload Definition
 
 Here are the key-value pairs in this event:
 
-| **Key or Key Type** | **Value Type** | **Optional** | **Description** |
-| :--- | :--- | :--- | :--- |
-| visitor\_identifiers | map | no | A map containing the following identifiers:keys = id typesvalues = correlating identifiers: |
-| authentication\_state | enum/string constant/number | yes |  |
+| **Key**               | **Value Type** | **Optional** | **Description**                                              |
+| :-------------------- | :------------- | :----------- | :----------------------------------------------------------- |
+| advertisingidentifier | String         | Yes          | The base URL passed as parameter to the `appendVisitorInfoForURL` (Android) / `appendToUrl` (iOS) API |
+
+### Event Data Example
+
+**Append to URL request**
+
+```json
+{
+    "baseurl": "https://example.com/path?query=parameter1&other=parameter2"
+}
+```
 
-## Visitor Get IDs Request
+### Event Data Example
 
-This event is generated by customers that use the `visitorGetIds` API.
+**Sync identifiers request**
 
-There are no key-value pairs in this event.
+```json
+{
+    "visitoridentifiers": {
+        "idtype1" : "id1",
+        "idtype2" : "id2"
+    },
+    "authenticationstate" : 1,
+    "forcesync" : false,
+    "issyncevent" : true
+}
+```
+
+### Event Data Example
```

Comment:
Remove this example.  Technically it would work using the `generic.identity` type with `issyncrequest` false, but that's not the intent of this API/event so let's not document it that way.